community/tcl/build

```diff
@@ -1,14 +1,21 @@
 #!/bin/sh -e
 
-IFS=. read -r ver1 ver2 _ < "${0%/*}/version"
+IFS=. read -r maj min _ < "${0%/*}/version"
 
-unix/configure \
+rm -rf pkgs/sqlite3*
+
+cd unix
+
+./configure \
     --prefix=/usr \
     --mandir=/usr/share/man \
     --enable-threads \
-    --enable-64bit
+    --enable-64bit \
+    --with-system-sqlite
 
 make
-make -j1 INSTALL_ROOT="$1" install install-private-headers
```

Comment:
Why was `-j1` removed? Unneeded?

Reply:
yes, unneeded

Reply:
I copied that -j1 from adelie, originally. It's probably because they target so many arches or? It's safe to remove for us, AFAIK. But there are sometimes weird make bugs that require -j1 for *some* and not *all* of us ... @dylanaraps Would you prefer I reintroduce the -j1 ?